main.cpp

```diff
@@ -73,12 +24,26 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+
+void printPair(){
```

Comment:
This `printPair` function is "production code", in the sense that it is meant to be used by consumers of this code.
Other functions in this file are "test code" - they help in proving our functionality, not functionality that's used by consumers.
Don't mix production and test code. People who want to use the printing function are forced to take the test code as well.